CHANGELOG.txt

```diff
@@ -164,6 +164,8 @@
             - The HTML id "editforms" has been removed.
             - The CSS class "table_detail_view" is not use anymore.
             - the CSS files "creme_core/css/blocks.css" have been removed from in the setting "CREME_CORE_CSS" (they will be removed in Creme 2.4)
+        # Fix issue in bulk edition of listview items (#170)
+            - Remove the 'BulkFieldSelectWidget' that is no longer useful and move its role to 'creme.lv_widget.EditSelectedAction' (JS).
```

Comment:
Rework the section "# In 'creme_core.forms.widgets' :" into "# In 'creme_core.forms' :", & move this line within.